react/.eslintignore

```diff
@@ -1,4 +1,5 @@
 dist/
 build/
 coverage/
-src/core-components/
\ No newline at end of file
+src/core-components/
+src/core-wrappers/
```

Comment:
This change is effectual. There are 15 lint problems in the new code. Acceptable.[^1]

<details><summary>Why didn't tup-ui fix these?</summary>

I never saw these lint issues on tup-ui via `nx format:check`, because… oh my gosh, [`libs/core-components/.eslintrc.json`](https://github.com/TACC/tup-ui/blob/v1.1.5/libs/core-components/.eslintrc.json#L3) and [`libs/core-wrappers/.eslintrc.json`](https://github.com/TACC/tup-ui/blob/v1.1.5/libs/core-wrappers/.eslintrc.json#L3) ignore everything too.

</details>

<details><summary>Linting the new <code>src/core-…</code> results in 15 problems.</summary>

```
/Users/wbomar/Code/hazmapper/react/src/core-components/Button/Button.test.jsx
  11:54  error  'getByTestId' is defined but never used  @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-components/DropdownSelector/DropdownSelector.jsx
  26:50  error  'className' is missing in props validation  react/prop-types

/Users/wbomar/Code/hazmapper/react/src/core-components/InfiniteScrollTable/InfiniteScrollTable.jsx
   84:11  error  Missing "key" prop for element in iterator  react/jsx-key
   86:15  error  Missing "key" prop for element in iterator  react/jsx-key
   95:13  error  Missing "key" prop for element in iterator  react/jsx-key
   98:19  error  Missing "key" prop for element in iterator  react/jsx-key
  129:22  error  'offset' is defined but never used          @typescript-eslint/no-unused-vars
  133:17  error  'row' is defined but never used             @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-components/SectionHeader/SectionHeader.jsx
  45:3  error  'isNestedHeader' is missing in props validation  react/prop-types

/Users/wbomar/Code/hazmapper/react/src/core-components/Sidebar/Sidebar.jsx
  15:65  error  'hidden' is defined but never used  @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-wrappers/FieldWrapperFormik/fields/FormikCheck.tsx
  10:3  error  'required' is defined but never used  @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-wrappers/FieldWrapperFormik/fields/FormikFileInput/FileDropzone.tsx
  81:37  error  'i' is defined but never used  @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-wrappers/Navbar/Navbar.tsx
  36:18  error  'end' is defined but never used  @typescript-eslint/no-unused-vars

/Users/wbomar/Code/hazmapper/react/src/core-wrappers/utils/withBuilder.tsx
  20:11  error  'slice' is defined but never used  @typescript-eslint/no-unused-vars
  21:11  error  'slice' is defined but never used  @typescript-eslint/no-unused-vars

✖ 15 problems (15 errors, 0 warnings)
```

</details>

[^1]: Several lint problems seem easy to fix. But, given that no one has been linting them thus far, I can't strongly argue to fix them immediately. It seems a `@tacc/core-…` package responsibility to lint it's own code.